### ISSUE TITLE
Reorganization the folders structure for testnet

### ISSUE THREAD

**Issue by magicstuff** [nebulasio/go-nebulas#55]
Hello!
What do you think about the reorganization of the folders structure and adding testnet  configuration files?
It seems to me so it would be convenient:
```
~/Go/src/github.com/nebulasio/go-nebulas $ tree testnet/
testnet/
├── conf
│   ├── config.conf
│   ├── ed25519key
│   └── genesis.conf
├── data.db
└── keydir
```
If you like it, I can do this.

**Comment by qywang2012** [comment]
yeah, It's ok. Maybe we should change ed25519key to `networkkey`, which is used as the p2p network private key. You can push a pull request do this change.

**Comment by magicstuff** [comment]
Sorry, I'm confused.

There is no info about generating by self a networkkey file for tesnet in this tutorial [[English] Nebulas 101 - 06 Testnet.md](https://github.com/nebulasio/wiki/blob/master/tutorials/%5BEnglish%5D%20Nebulas%20101%20-%2006%20Testnet.md). 
I copied the conf/network/ed25519key to testnet/conf/networkkey just for moving all testnet files in one place.
We can change or remove networkkey later, but I think we need to explain this in tutorial.


Now i did this https://github.com/magicstuff/go-nebulas/commits/develop, last 2 commits.
Explain me please, where I did wrong.

**Comment by qywang2012** [comment]
You can use neb subcommand `./neb network ssh-keygen` to generate the network key. I'm sorry that it's not write in tutorials.
If you have changed the position of the `networkkey`,  you should update the config in config file

```

network {
  listen: ["0.0.0.0:8680"]
  private_key: "conf/network/ed25519key" # you should change the key position for you self.
  network_id: 1
}
```
